README.md

```diff
@@ -24,14 +24,22 @@ sum_array
 */
 ```
 
-# Options
+# Configuration
 
-There are two `postgresql.conf` settings that must be configured:
+First, plrust must be configured as a `shared_preload_libraries` entry in `postgresql.conf`.  For example:
+
+```
+shared_preload_libraries = 'plrust'
+```
+
+Failure to do so will cause unexpected behavior around the DROP FUNCTION, DROP SCHEMA, and ALTER FUNCTION commands.
```

Comment:
I feel like we could have a script that people can run that preloads the required settings, but then they also have to manually set the `plrust.pg_config` and `plrust.work_dir` anyways...

Reply:
I don't think we could come up with a script that could guess any of this for any given installation, and that's most definitely not common practice in the postgres extension world.